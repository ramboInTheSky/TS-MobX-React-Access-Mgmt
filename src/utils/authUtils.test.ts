import {
    randomString,
    urlEncodeB64,
    bufferToBase64UrlEncoded,
    getConfig,
    sha256,
    debounce,
} from './authUtils'

describe('authUtils', () => {
    it('randomString', () => {
        expect(randomString).toMatchSnapshot()
    })

    it('urlEncodeB64', () => {
        expect(urlEncodeB64).toMatchSnapshot()
    })

    it('bufferToBase64UrlEncoded', () => {
        expect(bufferToBase64UrlEncoded).toMatchSnapshot()
    })

    it('sha256', () => {
        expect(sha256).toMatchSnapshot()
    })

    it('getConfig', () => {
        expect(getConfig).toMatchSnapshot()
    })

    it('debounce', () => {
        expect(debounce).toMatchSnapshot()
    })
})
