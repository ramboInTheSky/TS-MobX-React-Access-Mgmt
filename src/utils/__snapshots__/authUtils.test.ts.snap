// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`authUtils bufferToBase64UrlEncoded 1`] = `[Function]`;

exports[`authUtils debounce 1`] = `[Function]`;

exports[`authUtils getConfig 1`] = `[Function]`;

exports[`authUtils randomString 1`] = `[Function]`;

exports[`authUtils sha256 1`] = `[Function]`;

exports[`authUtils urlEncodeB64 1`] = `[Function]`;
